apps/src/netsim/NetSimVisualization.js

```diff
@@ -263,6 +266,15 @@ NetSimVisualization.prototype.onRemoteChange_ = function() {
   this.distributeForegroundNodes();
 };
 
+NetSimVisualization.prototype.onWireStateChange_ = function(
+  message,
+  fromRemote
```

Comment:
The contract for this event feels funny - if I'm receiving a "wire state change" event, I sort of assume the state is stored somewhere else, and I'd expect the parameter for the event to simply be the new state... `message` and `fromRemote` feel like implementation details. I'm not sure we actually store this wire state anywhere but the UI right now, though!  Maybe that's something we need to introduce.